cpp/src/parquet/arrow/reader_internal.h

```diff
@@ -76,7 +76,8 @@ class FileColumnIterator {
       return nullptr;
     }
 
-    auto row_group_reader = reader_->RowGroup(row_groups_.front());
+    int row_group_index = row_groups_.front();
+    auto row_group_reader = reader_->RowGroup(row_group_index);
```

Comment:
Does this change make any difference? I was expecting a type issue, but it seems like `row_groups_.front()` should be an `int` anyway.

Reply:
Without this change I was unable to determine at which row group index the failure occurred in the debugger. I can revert it but it seems useful to be able to know

Reply:
Ah I thought it was pop_front. I'll revert